diga-epa-client/src/test/java/com/oviva/epa/client/KonnektorServiceAcceptanceTest.java

```diff
@@ -213,24 +215,25 @@ private Document buildDocumentPayload(
             List.of(
                 new Author(
                     null,
-                    "Tester",
-                    "DIGA-Test",
+                    "Oviva Direkt for Obesity",
```

Comment:
Should we use the official german name, or this also official?
`Oviva Direkt für Adipositas`
https://diga.bfarm.de/de/verzeichnis/00872

Reply:
Yes, the current metadata is submitting correctly